README.md

```diff
@@ -1,2 +1,41 @@
-# Edge AI GStreamer Apps
-> Repository to host GStreamer based Edge AI applications for TI devices
+# Edge AI TIOVX Apps
+> Repository to host TIOVX based Edge AI applications for TI devices
+
+
+## Steps to run
```

Comment:
Why put the steps to run before putting the steps to compile ... if you only clone the repo and try to run, won't the executable not be there yet until you build?

Reply:
Updated README.md, considering all the suggestions above